pylsp/__init__.py

```diff
@@ -3,12 +3,27 @@
 
 import os
 import pluggy
+from . import _version
 from ._version import __version__
 
+def convertVersionInfo(version: str) -> (int, ..., str):
+    VERSION_INFO = version.split(".")
```

Comment:
Please rename this variable to `version_info` because it's a local one (as far as I can tell).

Reply:
Fixed.